### ISSUE TITLE
toggle didnt not work me as well i tried every thing i feel like i cant even move on from lecture  17 

### ISSUE THREAD

**Issue by sami313** [thedigicraft/Atom.CMS#110]
//index file
<?php  include('config/setup.php');  ?>

<!DOCTYPE html>
<html>
	<head>
		<title><?php echo $page['title'].' | '.$site_title; ?> </title>
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		
		<?php include('config/css.php'); ?>
		<?php include('config/js.php'); ?>
		
	
		
		
	</head>
	
	<body>
<div id="wrap">
	
 	<?php include(D_TEMPLATE.'/nav.php'); ?>  			
		
		
		
		<div class="container">
			
			<h1><?php echo $page['header']; ?></h1>
			
			<p><?php echo $page['body_formatted']; ?></p>
			
			<?php if(isset($_GET['debug']) == 1 ) { ?>
			
			<?php } ?>
		</div>
		
</div>
	
	
	<?php include(D_TEMPLATE.'/footer.php'); // page footer?>
	
	<div id="console-debug">
			<pre>
				 <?php print_r($page); ?> 
			</pre>
	</div>

	</body>
</html>

                                                                  //CSS FILE

<?php
// CSS:

?>

		
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">

			
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
		
		
		<link rel="stylesheet" href="https://code.jquery.com/ui/1.10.4/themes/smoothness/jquery-ui.css">
		
		
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
		
		<style>
			html,
			body{
				height: 100%;
				/* the html and body elements cannot have any padding or margin */
			}
			/* Wrapper for page content to push down footer BUT YET I DIDNT ADD WARP DIV FOR CONTENT*/
			#wrap{ 
				min-height: 100%;
				height: auto;
				/* Negative indent footer by its height */
				margin: 0 auto -60px;
				/* pad bottom by footer height */
				padding: 0 0 60px;
			}
			
			/* Set the fixed height of the footer here */
			#footer {
				height: 60px;
				background-color: #f5f5f5;
			}
			
			#btn-debug{
				
				position: absolute;
			
			}
			#console-debug{
				
				position: absolute;
				
			}
			
			
		</style>

                                        /////////////// JS FILE ////////////////////////////

		<?php
	//Javascript:
	
	
	
	?>
	
	
	
	
	
	
<script scr="https://code.jquery.com/jquery-1.10.2.min.js"></script>  
		
		
<script src="https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
		
		

<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
	
		<script>
			
			$(document).ready(funtion() {
				
				$("#console-debug").hide();
				
				
				
				$("#btn-debug").click(function() {
					
					$("#console-debug").toggle();
				
					});
				});
				
				
				
			
			
		</script>

**Comment by creptor** [comment]
I can't understand your problem, please be sure to specifi it well and to report any errors found on the page AND then i may ask you for some files, the lines of code that you pasted are hard to work for me :cry:

**Comment by sami313** [comment]
My problem is that my < div id="console-debug"> test debug window </div> that test debug window never hide it self its always on screen like hardcode now pleasw tell me sloution i tired my best :(

**Comment by creptor** [comment]
when you need it to be hidden?

**Comment by creptor** [comment]
don't worry I got it...
please giveme your js.php file please

**Comment by sami313** [comment]
<?php
	//Javascript:
	
	
	
	?>
	
	
	
	
			
			
		<script scr="https://code.jquery.com/jquery-1.10.2.min.js"></script>  
				
				
		<script src="https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
				
				
		
		<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
			
		<script>
			
			$(document).ready(funtion() {
				
				$("#console-debug").hide();
				
				
				
				$("#btn-debug").click(function() {
					
					$("#console-debug").toggle();
				
					});
				});
				
				
				
			
			
		</script>

**Comment by sami313** [comment]
![home page sami site](https://cloud.githubusercontent.com/assets/11160392/6747160/02a8df3c-cef7-11e4-9ce1-a9bd0eb3de32.png)

**Comment by sami313** [comment]
as you can see the i never even click debug button its always there as my page load ..yes i am working local host i tried my best to locate the problem please help me

**Comment by creptor** [comment]
usin https could make the browsers deny info if you do not have a valid certificate, try to use http and see if the problem is solved. if it's not that just let me know.

**Comment by sami313** [comment]
i did all the stuff ..should i download jquery and add them locally ??? please help me out i really want to listen your all lecture but with this 17 i am stuck :( thanks for your support

**Comment by sami313** [comment]
can you send me your email or any thing ...if you want i can mail you rar copy of code so you have load it and run it

**Comment by sami313** [comment]
![home page sami site](https://cloud.githubusercontent.com/assets/11160392/6765087/99a90f78-cff2-11e4-9ff7-72aeb71cefca.png)

**Comment by sami313** [comment]
its not hiding (window test debug)....i just want to bang my head against wall now :D  i have already added every library locally so that http or https things get over ..man please please you have to help me on that i have already killed my 3 days on it ..i need to gear up

**Comment by sami313** [comment]
<?php
	//Javascript:
	
	
	
	?>
	
	
	
	
			
			
		<script scr="js/jquery-2.1.3.min.js"></script>  
				
				
	<script src="js/jquery-ui.min.js"></script>
				
				
		
	<script src="dist/js/bootstrap.min.js"></script>
			
		<script>
			
			$(document).ready(function() {
				
				$("#console-debug").hide();
				
				
				
				});
				
				
				
			
			
		</script>

**Comment by JasonMate** [comment]
Try this?
<script src="
https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>

**Comment by sami313** [comment]
i have tried that as well its not working

**Comment by sami313** [comment]
yahhhhhhhhhhhhhhhhhhhhhhhhhhhhhooooooooooooooooooooooooooooo man finally i did it :dancer:

**Comment by sami313** [comment]
<script type="text/javascript">
		
		$(document).ready(function() {
			
			$("#console-debug").hide();
			
				$(window).load(function(){ 
				
					$("#btn-debug").click(function(){
						
						$("#console-debug").slideToggle(400);
						
					});	
			
		
			});
		
		
		});
		
	</script>

**Comment by sami313** [comment]
<script type="text/javascript">
		
		$(document).ready(function() {
			
			$("#console-debug").hide();
			
				$(window).load(function(){ 
				
					$("#btn-debug").click(function(){
						
						$("#console-debug").slideToggle(400);
						
					});	
			
		
			});
		
		
		});
		
	</script>

**Comment by sami313** [comment]
this way it will work only if you write the code this way 
<script type="text/javascript">
		
		$(document).ready(function() {
			
			$("#console-debug").hide();
			
				$(window).load(function(){ 
				
					$("#btn-debug").click(function(){
						
						$("#console-debug").slideToggle(400);
						
					});	
			
		
			});
		
		
		});
		
	</script>

**Comment by sami313** [comment]
<?php
	//Javascript:
	
	
	
	?>
	
	
	
	
			
			
		<script scr="js/jquery-2.1.3.min.js"></script>  
				
				
	<script src="js/jquery-ui.min.js"></script>
				
				
		
	<script src="dist/js/bootstrap.min.js"></script>
			
		<script type="text/javascript" src="js/jquery-2.1.3.min.js"	></script>
	
	<script type="text/javascript">
		
		$(document).ready(function() {
			
			$("#console-debug").hide();
			
				$(window).load(function(){ 
				
					$("#btn-debug").click(function(){
						
						$("#console-debug").slideToggle(400);
						
					});	
			
		
			});
		
		
		});
		
	</script>

**Comment by JasonMate** [comment]
Happy dance. Party at Alan's house!

**Comment by sami313** [comment]
yah man i was stuck on that from last three days .... :dancers:  and where is alan's house i am not familiar with that tearm :dancer:

**Comment by creptor** [comment]
nice job finding a way to resolve your problem :smile:
